### ISSUE TITLE
[BUG] Release builds failing on Android with VisionCamera v4

### ISSUE THREAD

**Issue by mfkrause** [nonam4/react-native-vision-camera-face-detector#47]
**Describe the bug**
Release builds for Android are failing using VisionCamera v4 and this plugin.

**To Reproduce**
1. Install react-native-vision-camera@^4
2. Install react-native-vision-camera-face-detector@latest
3. Import both into the app
4. Start a release build for Android (in my case, using Expo EAS local – shouldn't matter, though)

**Expected behavior**
Builds should succeed.

**Logs and Screenshots**
```
[...]
[RUN_GRADLEW] > Task :react-native-vision-camera:compileReleaseJavaWithJavac
[RUN_GRADLEW] > Task :react-native-vision-camera:bundleLibCompileToJarRelease
[RUN_GRADLEW] > Task :expo-modules-core:compileReleaseJavaWithJavac
[RUN_GRADLEW] Note: Some input files use or override a deprecated API.
[RUN_GRADLEW] Note: Recompile with -Xlint:deprecation for details.
[RUN_GRADLEW] Note: Some input files use unchecked or unsafe operations.
[RUN_GRADLEW] Note: Recompile with -Xlint:unchecked for details.
[RUN_GRADLEW] > Task :expo-modules-core:bundleLibCompileToJarRelease
[RUN_GRADLEW] > Task :react-native-gesture-handler:compileReleaseJavaWithJavac
[RUN_GRADLEW] > Task :repo_vision-camera-face-detector:compileReleaseKotlin FAILED
[RUN_GRADLEW] > Task :expo-application:compileReleaseKotlin
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPlugin.kt:15:28 Unresolved reference: frameprocessor
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPlugin.kt:16:28 Unresolved reference: frameprocessor
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPlugin.kt:17:28 Unresolved reference: frameprocessor
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPlugin.kt:21:10 Unresolved reference: VisionCameraProxy
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPlugin.kt:23:5 Unresolved reference: FrameProcessorPlugin
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPlugin.kt:234:3 'callback' overrides nothing
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPlugin.kt:235:12 Unresolved reference: Frame
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPluginPackage.kt:7:28 Unresolved reference: frameprocessor
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPluginPackage.kt:12:7 Unresolved reference: FrameProcessorPluginRegistry
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPluginPackage.kt:13:9 Cannot infer a type for this parameter. Please specify it explicitly.
[RUN_GRADLEW] e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/0d23399a-98e6-4e6a-bf3b-88eda06710fe/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPluginPackage.kt:13:16 Cannot infer a type for this parameter. Please specify it explicitly.
[...]
[RUN_GRADLEW] FAILURE: Build failed with an exception.
[RUN_GRADLEW] * What went wrong:
[RUN_GRADLEW] Execution failed for task ':repo_vision-camera-face-detector:compileReleaseKotlin'.
[RUN_GRADLEW] >
[RUN_GRADLEW] A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
[RUN_GRADLEW]    > Compilation error. See log for more details
[RUN_GRADLEW] * Try:
[RUN_GRADLEW] > Run with --stacktrace option to get the stack trace.
[RUN_GRADLEW] > Run with --info or --debug option to get more log output.
[RUN_GRADLEW] > Run with --scan to get full insights.
[RUN_GRADLEW] > Get more help at https://help.gradle.org.
[RUN_GRADLEW] BUILD FAILED in 2m 29s
```

**Smartphone (please complete the following information):**
 - Device: not relevant
 - OS: Android
 - Vision Camera version: ^4
 - Face detector version: latest

**Additional context**
Related to this issue in the VisionCamera repo (which has to do with the vision-camera-resize-plugin): https://github.com/mrousavy/react-native-vision-camera/issues/2786

The compiler errors above can be fixed by replacing the `import com.mrousavy.camera.frameprocessor.*` imports with `import com.mrousavy.camera.frameprocessors.*`. However, this leads to a new error:

```
e: file:///private/var/folders/h5/w2kdxkb56x9b1fvcqxrg4kt40000gn/T/eas-build-local-nodejs/69d76b88-a629-4cf1-bcc8-89c48213aaf8/build/apps/mobile/node_modules/react-native-vision-camera-face-detector/android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPlugin.kt:252:36 Unresolved reference: toDegrees
```

It seems like `frame.orientation` has no function property `toDegrees()` anymore (used in `android/src/main/java/com/visioncamerafacedetector/VisionCameraFaceDetectorPlugin.kt`). Taking a look at the new implementation, there's indeed no `toDegrees()` function anymore: https://github.com/mrousavy/react-native-vision-camera/blob/0e5a888c00fa4226d8acf7df6d56689d76e8ea71/package/android/src/main/java/com/mrousavy/camera/core/types/Orientation.kt

**Comment by nonam4** [comment]
Use `1.6.0` version instead. Also see `1.6.1` release notes.

**Comment by mfkrause** [comment]
@nonam4 Ah, my bad. I messed up the version in my package.json and thought I already had the latest version. Sorry about that. :)

**Comment by nonam4** [comment]
@mfkrause take a look into `1.6.1` release notes. There’s a linked issue related to worklets-core in released app.

**Comment by luicfrr** [comment]
I just released 1.6.2 version 🥳🎉 It is using latest vision-camera and worklets-core and it's not crashing on release mode. Can you please test it and give me a feedback? Thanks
